### ISSUE TITLE
Please consider not adding extra global margins to sidebars

### ISSUE THREAD

**Issue by apoapostolov** [Rayuaz/twilight-ui#26]
This doesn't contribute much to the design and it makes some posts get even longer vertically due to wasted space. 

![image](https://user-images.githubusercontent.com/4083812/125966289-d82424a8-7223-4abb-90d9-f682cfa65193.png)

**Comment by Rayuaz** [comment]
This a stylistic choice. I feel that the default Foundry interface lacks a lot of white space. I might add a compact version of the UI in the future, and if I do, I'll remove the extra padding.

**Comment by apoapostolov** [comment]
Okay, I will have to Custom CSS this for now. Since this issue will stay unresolved until Settings are introduced, maybe keep it open or give it a Future Improvement tag, to make sure these aren't fully forgotten?

**Comment by Rayuaz** [comment]
Fair point. I'm still figuring out how to use this thing. Btw, you might want to either add a `padding-top: 15px` to windows with tabs or `margin-top: 0` to `nav.sheet-tabs:first-child`.

**Comment by apoapostolov** [comment]
Thank you!
